build/teamcity/internal/release/process/build-cockroach-release-per-platform.sh

```diff
@@ -68,7 +68,7 @@ if [[ $platform == "linux-amd64" || $platform == "linux-arm64" || $platform == "
     --ungzip \
     --ignore-zeros \
     --strip-components=1
-  cp --recursive licenses "build/deploy-${platform}"
+  cp LICENSE THIRD-PARTY-NOTICES.txt "build/deploy-${platform}"
```

Comment:
```suggestion
  cp LICENSE licenses/THIRD-PARTY-NOTICES.txt "build/deploy-${platform}"
```

I think this also should be applied to the master branch and other backports.

Reply:
🤦 right. Will make that change.